### ISSUE TITLE
Commit Message Description In CONTRIBUTING.MD

### ISSUE THREAD

**Issue by RishabhBhatnagar** [spdx/tools-python#151]
CONTRIBUTING.MD file should also describe how a commit message should be written or rather what a commit message should be.
There is a [recent commit](https://github.com/spdx/tools-python/commit/209a92563ea321403e0e8abc2308644367d73996) in a PR whose title doesn't conclude the changes made and looks like a commit description being used as a commit title.

A good commit message will foster traceability in a repository.

**Comment by harshkasat** [comment]
@meretp Is the issue still open? bcoz it label as tech debt

**Comment by meretp** [comment]
Yes, it is still open. As with #113 we used this label to categorize this as something that is not urgent but rather a nice-to-have. One could potentially find a better label for this
